x-pack/legacy/plugins/uptime/public/components/functional/alerts/toggle_alert_flyout_button.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiButtonEmpty, EuiContextMenuItem, EuiContextMenuPanel, EuiPopover } from '@elastic/eui';
+import React, { useState } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { useKibana } from '../../../../../../../../src/plugins/kibana_react/public';
+
+interface Props {
+  setAlertFlyoutVisible: (value: boolean) => void;
+}
+
+export const ToggleAlertFlyoutButtonComponent = ({ setAlertFlyoutVisible }: Props) => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const kibana = useKibana();
+
+  return (
+    <EuiPopover
+      button={
+        <EuiButtonEmpty
+          data-test-subj="xpack.uptime.alertsPopover.toggleButton"
+          iconType="arrowDown"
+          iconSide="right"
+          onClick={() => setIsOpen(!isOpen)}
+        >
+          <FormattedMessage
+            id="xpack.uptime.alerts.toggleAlertFlyoutButtonText"
+            defaultMessage="Alerting"
```

Comment:
```suggestion
            defaultMessage="Alerts"
```

Reply:
Addressed in https://github.com/elastic/kibana/pull/57919/commits/2519a8ce744e8b686199e692437b132c9d61eec1.